File_handling/fileOperations.test.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+
+const {persons, saveDataToFile} = require('./fileOperations.js')
+
+test('Reading data from file', () => {
+    const readMe = 'persons.json'
+    const testData = fs.readFileSync(readMe, 'utf-8')
+    const expectedData = JSON.parse(testData)
+    expect(persons).toEqual(expectedData)
+})
+
+test('Saving data to file', () => {
+    const readMe2 = 'persons2.json'
```

Comment:
Be sure to add the generated file to gitignore to avoid commuting it

Reply:
I don't understand...If I don't commit the file fileOperations.test.js, how will you access the unit tests?

Reply:
You are creating a file during the tests, right? Add the generated files to gitignore because they are not necessary and are only temporary.

Reply:
ok.